client/src/AppBundle/Resources/assets/javascripts/modules/textarea-autosize.js

```diff
@@ -4,14 +4,14 @@
 // Note that associated styles live under _.forms.scss
 
 module.exports = function (containerSelector) {
-  var textArea = $(containerSelector).find("[class*='js-auto-size'] textarea")
+  const textArea = $(containerSelector).find("[class*='js-auto-size'] textarea")
 
   textArea.on('keyup input paste change', function (event) {
-    var $this = $(event.target)
+    let $this = $(this)
+    let initialHeight = $this.height()
```

Comment:
Stealing from `npm run lint`:
```suggestion
    const $this = $(this)
    const initialHeight = $this.height()
```